docs/en/writing-running-appium/caps.md

```diff
@@ -137,6 +137,8 @@ Appium tries to guess the version of the WebView based on the version of the cor
 |`buildToolsVersion`| Specify the Android `build-tools` version to be something different than the default, which is to use the most recent version. It is helpful to use a non-default version if your environment uses alpha/beta build tools. | e.g. `'28.0.3'` |
 |`androidNaturalOrientation`| Allow for correct handling of orientation on landscape-oriented devices. Set to `true` to basically flip the meaning of `PORTRAIT` and `LANDSCAPE`. Defaults to `false` | `true`, `false` |
 | `enforceAppInstall` | By default application installation is skipped if newer or the same version of this app is already present on the device under test. Setting this option to `true` will enforce Appium to always install the current application build independently of the currently installed version of it. Defaults to `false`. | `true` , `false` |
+| `ignoreHiddenApiPolicyError` | Ignores `Security exception: Permission denial` alert to proceed the create session process since Appium 1.18.0+.. The error happens when Appium try to relax [hidden API policy](https://developer.android.com/distribute/best-practices/develop/restrictions-non-sdk-interfaces#how_can_i_enable_access_to_non-sdk_interfaces) for Appium feature. But some devices restrict the permission. Defaults to `false`. | `true`, `false` |
+| `mockLocationApp` | Set an arbitrary mock location provider as bundle id for a real device since Appium 1.18.0+. Defaults to Appium's settings app. If the value is `null`, Appium will skip the location provider setup. | `null`, `io.appium.settings`, `example.your.app` |
```

Comment:
Sets the package identifier of the app, which is used as a system mock location provider since Appium...

Reply:
The capability is ignored on emulators and Android versions below six (API level 23)

Reply:
If the value is set to `null` or an empty string, then Appium will skip the mocked location provider setup procedure

Reply:
Defaults to Appium Settings package name (`io.appium.settings`)